cluster/gce/config-default.sh

```diff
@@ -74,6 +74,14 @@ DNS_SERVER_IP="10.0.0.10"
 DNS_DOMAIN="cluster.local"
 DNS_REPLICAS=1
 
+# Optional: Create autoscaler for cluster's nodes.
```

Comment:
How about adding the same options to config-test.sh? Alternatively we may want to factor the common configs into config-common.sh